pages/belt_docs/list.mdx

```diff
@@ -461,28 +1177,89 @@ let keepMapU: (t('a), [@bs] ('a => option('b))) => t('b);
 let keepMap: (t('a), 'a => option('b)) => t('b);
 ```
 
-## partitionU
+```reason
+keepMap(someList, f);
+```
+
+Applies `f` to each element of `someList`. If `f(xi)` returns `Some(value)`, then `value` is _kept_ in the resulting list.
+If `f(xi)` returns `None`, the element is _not_ retained in the result.
+
+**Examples**
+
+```reason
+let isEven = x => x mod 2 == 0;
+
+[1, 2, 3, 4]
+->Belt.List.keepMap(x =>
+    if (isEven(x)) {
+      Some(x);
+    } else {
+      None;
+    }
+  ); /* [2, 4] */
 
+[Some(1), Some(2), None]->Belt.List.keepMap(x => x); /* [1, 2] */
 ```
-let partitionU: (t('a), [@bs] ('a => bool)) => (t('a), t('a));
+
+## keepMapU
+
+```
+let keepMapU: (t('a), [@bs] ('a => option('b))) => t('b);
 ```
 
+Uncurried version of [keepMap](#keepMap).
+
 ## partition
 
 ```
 let partition: (t('a), 'a => bool) => (t('a), t('a));
 ```
 
+```reason
+partition(someList, p);
```

Comment:
Here it's called `p`, but sometimes we use `f`... would be interesting if we should align that somehow? Or is `predicate` (p) semantically different than in cases where we use `f`?

Reply:
I'm not completely sure to be honest. Whenever I got hesitant I tried to fall back on the original documentation and perhaps change the wording a bit. 

At first I changed `p` to `predicate` but reverted once I got stuck in e.g.:

"by the predicate `predicate`"

Open to suggestions here!

Reply:
We have a weird mix of "mathematical slang" (let foo be a value within l, where l is a list of foo) ... maybe instead of `p` we could go for `pred`? Or we can just keep it for now and think about that later on